.github/workflows/screener-run.yml

```
@@ -0,0 +1,183 @@
+name: Screeners workflow
```

```suggestion
name: Screener run
```

nit: seems more appropriate